os/hal/ports/AT32/AT32F41x/hal_efl_lld.h

```diff
@@ -0,0 +1,122 @@
+/*
+    ChibiOS - Copyright (C) 2006..2018 Giovanni Di Sirio
+    ChibiOS - Copyright (C) 2023 HorrorTroll
+    ChibiOS - Copyright (C) 2023 Zhaqian
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+/**
+ * @file    hal_efl_lld.h
+ * @brief   AT32F41x Embedded Flash subsystem low level driver header.
+ *
+ * @addtogroup HAL_EFL
+ * @{
+ */
+
+#ifndef HAL_EFL_LLD_H
+#define HAL_EFL_LLD_H
+
+#if (HAL_USE_EFL == TRUE) || defined(__DOXYGEN__)
+
+/*===========================================================================*/
+/* Driver constants.                                                         */
+/*===========================================================================*/
+
+/*===========================================================================*/
+/* Driver pre-compile time settings.                                         */
+/*===========================================================================*/
+
+/**
+ * @name    AT32F41x configuration options
+ * @{
+ */
+/**
+ * @brief   Suggested wait time during erase operations polling.
+ */
+#if !defined(AT32_FLASH_WAIT_TIME_MS) || defined(__DOXYGEN__)
+#define AT32_FLASH_WAIT_TIME_MS            1
```

Comment:
Typical value in DS of at32f415 is 8.
![image](https://github.com/qmk-arterytek/ChibiOS-Contrib/assets/80608733/ff0e9059-8af6-4c0b-96e2-ada82c16f0dd)

Reply:
Yeah, this one is really strange. On the STM32F1x at chibios repo, they wrote 1 ms. But on the datasheet, it should be 40 ms. So I've just leave it like that, I'll update it to 8 ms now

Reply:
It seems that it's just a query operation, so it doesn't important.

Reply:
What else do I need for other periph lld? Maybe I can pr them to your branch this weekend, or you can sort them out.

Reply:
Maybe you can do the DMA thing? I remember it have flexible DMA and solid DMA.

Reply:
Okay,my pleasure.The initialization code is in pr#1, and if you have any questions, welcome to throw up the review conversation.

Reply:
Thank you, I really appreciate. If anything I need, I'll asking you about it ☺️